core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -172,6 +172,8 @@ private[ui] class AllJobsPage(parent: JobsTab) extends WebUIPage("") {
 
     val jobEventJsonAsStrSeq = makeJobEvent(jobs)
     val executorEventJsonAsStrSeq = makeExecutorEvent(executors)
+    val offset = TimeZone.getTimeZone(System.getProperty("user.timezone"))
+      .getOffset(System.currentTimeMillis()) / 1000 / 60
```

Comment:
Since `getRawOffset` does not consider Daylight Saving Time, I needed to use the above.